src/status_im/ui/screens/profile/contact/views.cljs

```diff
@@ -229,4 +228,4 @@
          :render-data               {:chat-id current-chat-id
                                      :profile true}
          :render-fn                 status.views/render-message
-         :data                      messages}]])))
+         :data                      (when (:added? contact) messages)}]])))
```

Comment:
we should see messages for not added contacts

Reply:
Hi @flexsurfer, Thank you for reviewing the PR and confirming desired behavior. (changes reverted)